testutil/singlevm/cleanup.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
-if [ ! -z $1 ]; then
-    hosts_file_backup=$1
-else
-    . ~/local/demo.sh
-    hosts_file_backup=$HOSTS_FILE_BACKUP
-fi
+. ~/local/demo.sh
```

Comment:
@rbradford leaving it in place now, as it may be needed if we want to use the image service interface to cleanup the images vs just rm as we do today.